HTMLSerializer.js

```diff
@@ -347,6 +413,21 @@ var HTMLSerializer = class {
   }
 
   /**
+   * Create a function that will generate unique strings which can be used as
+   * ids.
+   *
+   * @return {Function} A funtion that generates a unique string each time it is
+   *     called.
+   */
+  generateIDGenerator() {
```

Comment:
Hmm.  Good point.  Two options come to mind:
1.  Keep track of id's, and make sure there are no duplicates.  However, if I later encounter an id that I've already inserted, I would have to go back and change it.
2.  Strip away all id attributes except for the ones I am inserting, because I'm already stripping away all javascript, and style elements.  Would the ids be used for anything else?

Does either option strike you as better?  Option 2 seems simpler to me, but I don't know if there is some reason it can't be done.